cmd/kpromo/cmd/pr/pr.go

```diff
@@ -158,6 +158,9 @@ func init() {
 	}
 }
 
+// Another NOTE by @H0llyW00dzZ: This function is overly complex and difficult to read, both for humans and machines.
+// It would be better to split it into smaller functions, as this is Go, not Python, and Go
+// encourages simpler, more readable code.
```

Comment:
Can we drop this comment in favor of an issue? Issues are much easier to track than code comments. Also, issues are much easier to discover, giving a better chance to someone to find it and eventually help out.

Reply:
done 
- kubernetes-sigs#1177